metadata-ingestion/src/datahub/configuration/common.py

```diff
@@ -104,7 +101,7 @@ class OauthConfiguration(ConfigModel):
 
 
 class AllowDenyPattern(ConfigModel):
-    """A class to store allow deny regexes"""
```

Comment:
The plural `regexes` was correct. It is not `regexs`. Please change back in all places